ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -41,6 +42,25 @@ struct ToDoListView: View {
                         }
                     }
                     .listStyle(.plain)
+                    .padding(.bottom, addButttonSize)
```

Comment:
Just checked the UI. the padding clips list content 
<img width="257" alt="image" src="https://user-images.githubusercontent.com/24592433/207405179-809ee8a9-230f-4182-bb3b-17256fb56aa0.png">
and this should not happen, the content should be visible all the way to the bottom edge of the screen like this:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/24592433/207405379-77d5324f-ea87-465f-b3ae-d9f018eeb077.png">
Previously I suggested trying to use EmptyView() as the last cell of the List. But EmptyView doesn't work. It looks like EmptyView always has zero frame. In that case Color.clear may be used with height equals to the add button height

Reply:
This has been fixed.